src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private Random random = new Random();
+    private String randomColor;
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        switch (figureNumber) {
+            case 1:
```

Comment:
start from 0, case 4 and default should return the same random figure

Reply:
cases formated accordingly